package.json

```diff
@@ -43,5 +43,8 @@
     "dom-serializer": "^0.1.0",
     "htmlparser2": "^3.9.1",
     "object-assign": "^4.1.0"
+  },
+  "devDependencies": {
+    "fs": "0.0.2"
```

Comment:
What's the distinction between dependencies and devDependencies on this project... usually all the gulp stuff would only be needed in dev but the thing is gulp.